package.json

```diff
@@ -221,7 +221,7 @@
     "source-map-support": "0.5.21",
     "terser": "5.18.2",
     "tslib": "2.6.0",
-    "typescript": "5.1.5",
+    "typescript": "5.1",
```

Comment:
Why this change?

Reply:
ESLint (via pre-commit) had a warning along the lines that it's only supported up to 5.1 (ESTree).